### DISCUSSION TITLE
What is File Watcher

### DISCUSSION THREAD

**Discussion by levyra** [Natizyskunk/vscode-sftp#190] [Q&A] [answered]
File Watcher is listed as a feature in VS Code:

Does file watcher alert you to others changing files on the server?
If so, how is it enabled?

**Comment by andremuller** [comment, marked as answer]
AFAIK, it watches your local filesystem files so, when the local files are changed, the upload/delete operation is automatically executed according to your settings.

So if you change a file outside VS Code (using a CLI git interface for instance) it will be synced to the server.

https://github.com/Natizyskunk/vscode-sftp/wiki/Common-Config#watcher

**Reply by Natizyskunk** [reply]
This is exactly how it work.
